README.md

```diff
@@ -13,20 +13,5 @@ one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,
 
 Proof
 
-- lets assume we have two graphs, $G_1=(V_1 , E_1)$ and $G_2=(V_2 , E_2)$. $G_1$ and $G_2$ are isomorphic such that there exists a function $f: V_1 \rightarrow V_2$ such that $(u,v)
-\in E_1$ iff $(f(u),f(v)) \in E_2$.
-
-- if the two graphs are isomorphic and they have the same number of nodes and edges, then ...
-
-- $(\forall v \in V) \sum_{v \in V_1} deg(v) \implies \sum_{f(v) \in V_2} deg(f(v))$
-
-- $\sum_{v \in V_1} deg(v) \implies \sum_{f(v) \in V_2} deg(f(v)) $
-
-- $(deg(v) = deg(f(v))) \implies  2(E_1) = 2(E_2)$
-
-- now lets assume $G_1$ is not completley connected such that $(u, v) \notin E_1$.
-
-- $(u,v) \notin E1 \implies (f(u),f(v)) \notin E_2$ (i.e. we show that if $G_1$ is not completley connected, then $G_2$ must also not be completley connected)
-
-- this shows that if $G_1$ and $G_2$ are isomorphic, there can exist a disconnected node in $G_1$ as long as there also exists a corresponding disconnected node in $G_2$ such that $(u,v) \notin E1 \implies (f(u),f(v)) \notin E_2$ (i.e. $G_1$ and $G_2$ do not have to be completley connected in order be isomorphic)
+- We have two graphs $A=(V_1 , E_1)$ and $B=(V_2 , E_2)$ that are isomorphic to eachother and to themselves (i.e. they are both one to one and onto). There exists a disconnected node in $A$ therefore $A$ is not completley connected. And since $A$ and $B$ are isomorphic and there is a one to one and onto mapping, then there also must exist a disconnected node in $B$, therefore if two graphs $A$ and $B$ are isomorphic they do not have to be completely connected.
```

Comment:
You still need to show that they can be isomorphic if they're not completely connected -- here you're just assuming both things.